<?xml version="1.0" encoding="utf-8"?>
<launch>

    ### ROSBRIDGE ###
    <include file="$(find rosbridge_server)/launch/rosbridge_websocket.launch"/>

    ### Semantic_maping Virtual###
    <node name="semantic_mapping" pkg="vimantic" type="ViMantic_RobotAtVirtualHome_mxnet_node.py" output="screen">
        <param name="topic_result" value="ViMantic/SemanticObjects"/>
        <param name="topic_virtualCameraRGBD" value="RobotAtVirtualHome/VirtualCameraRGBD"/>
        <param name="topic_republic" value="ViMantic/ToCNN"/>
        <param name="topic_cnn" value="rr_mxnet/detections"/>
        <param name="debug" value="true"/>
    </node>

    ### RVIZ ###
    <node name="rviz_player" pkg="rviz" type="rviz"
          args="-d $(find vimantic)/launch/ViMantic_RobotAtVirtualHome_mxnet.rviz"/>


</launch>
